services/skipchain/skipchain.go

```diff
@@ -320,7 +320,7 @@ func (s *Service) startBFTSignature(block *SkipBlock) error {
 		if err := block.BlockSig.Verify(network.Suite, el.Publics()); err != nil {
 			return errors.New("Couldn't verify signature")
 		}
-	case <-time.After(time.Second * 3):
+	case <-time.After(time.Second * 10):
```

Comment:
Why not. If you think that makes sense. I'm just afrait 10sec is too short, but I would not know the best timeout to put; so 1mn seems reasonably long enough for me.